README.md

```diff
@@ -14,54 +13,39 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const id = '42'
-const postUrl = urlJoin('/posts', id, '/metadata/');
-// => "/posts/42/metadata"
+// Create a Full URL
```

Comment:
I think we can omit this comment, the code should be pretty self-explanatory in this small example.